File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/dataclasses/dataClassDocs/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.dataclasses.dataClassDocs.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String Search = "search";

    public static class SEARCH_INPUT_ARGUMENT {
      public static final String MovieFilter = "movieFilter";
    }
  }

  public static class MOVIE {
    public static final String TYPE_NAME = "Movie";

    public static final String Title = "title";
  }

  public static class MOVIEFILTER {
    public static final String TYPE_NAME = "MovieFilter";

    public static final String TitleFilter = "titleFilter";
  }
}

File: graphql-dgs-codegen-core/src/test/resources/kotlin2/dataClassDocs/expected/DgsConstants.kt
package kotlin2.dataClassDocs.expected

import kotlin.String

public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val Search: String = "search"

    public object SEARCH_INPUT_ARGUMENT {
      public const val MovieFilter: String = "movieFilter"
    }
  }

  public object MOVIE {
    public const val TYPE_NAME: String = "Movie"

    public const val Title: String = "title"
  }

  public object MOVIEFILTER {
    public const val TYPE_NAME: String = "MovieFilter"

    public const val TitleFilter: String = "titleFilter"
  }
}
